src/logging/Logger.h

```diff
@@ -5,6 +5,12 @@
 #include "debug.h"
 #include <Arduino.h>
 
+// Avoid circular dependancy on Network - Logger - Network
+#include <WiFiClient.h>
+namespace Network {
```

Comment:
Actually, we have a file for this: `GlobalVars.h`

Reply:
Most of the modules in `GlobalVars.h` reference Logger in some way, so including `GlobalVars.h` into `Logger.h` would raise error about using Logger before defining them. Couldn't throw network into `Logger.cpp` as there's a template function using network.

Reply:
I see